recipes/go/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set version = "1.4.2" %}
+
+package:
+  name: go
+  version: {{ version }}
+
+source:
+  fn: go-{{ version }}.tar.gz
+  url: https://storage.googleapis.com/golang/go{{ version }}.src.tar.gz
+  sha1: 460caac03379f746c473814a65223397e9c9a2f6
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Meh :-/

-> https://www.appveyor.com/docs/installed-software#go
-> https://golang.org/doc/install/source

Hm, that doesn't look so difficult...